das_api/src/api/api_impl.rs

```diff
@@ -409,6 +418,8 @@ impl ApiContract for DasApi {
             page.map(|x| x as u64),
             before.map(|x| bs58::decode(x).into_vec().unwrap_or_default()),
             after.map(|x| bs58::decode(x).into_vec().unwrap_or_default()),
+            tree,
```

Comment:
In terms of readability, it would be cleaner if you get the leaf_id / tree_id before this and then rename the method called `get_signatures_for_leaf`

Reply:
Wouldn't that break this endpoint for existing users who are using array params?

Reply:
Ah sorry I misread